tools/tripal/db_index.xml

```diff
@@ -110,6 +119,18 @@
             <option value="path_hierarchy">path_hierarchy</option>
         </param>
         <param name="queues" label="Queues" argument="queues" type="integer" value="10" help="Number of indexing task queues" />
+        <conditional name="expose">
+            <param name="do_expose" type="select" label="Expose the index to other websites (read-only)">
+                <option value="yes" selected="true">No</option>
+                <option value="no">No</option>
+            </param>
+            <when value="no"/>
+            <when value="yes">
+                <param name="index_url" label="Index Url" argument="--index_url" type="text" help="In order for other sites to link back to your results page, you must specify a path where the form for this index can be reached" />
```

Comment:
It would make sense yes, does this validator already exist? I'm not sure this option will ever be used, so not top priority for me if it needs to be written :)

Reply:
I do not think such a validator exists, can you create a galaxy issue, maybe someone in the future can implement this.

Reply:
Yep, just created https://github.com/galaxyproject/galaxy/issues/7408